components/footer/_footer.scss

```diff
@@ -1,13 +1,40 @@
 .footer {
   padding: 2rem 0;
+  padding-bottom : 4rem;
   margin-top: 2rem;
-  background-color: $color-light-grey;
-  color: $color-very-dark-grey;
-}
+  background-color: $color-very-dark-grey;
+  color: $color-light-grey;
+
+  .footer__heading {
```

Comment:
Is there any reason to nest these rules? It creates an extra, potentially unnecessary, level of specificity.

Reply:
This is much needed I think to help guide ONS developers when following BEM methods. Having seen many examples of invalid BEM usage with elements used outside of blocks (standard BEM allows this). Elements should only exist in the context of a block and this better enforces that rule.